### PR TITLE
[mobile] Fix the theme part of the dialog

### PR DESCRIPTION
**Pull request by DohyungLim**
- Chnage the button font siz used in the dialog footer.
- Change the secondary text color.

### CODE REVIEW

**Review by HunseopJeong — APPROVED** [review]
LGTM

File: src/css/profile/mobile/common/popup.less [hunk 1]
```diff
@@ -192,7 +195,7 @@
 		height: auto;
 		margin-top: 8 * @px_base;
 		margin-left: 8 * @px_base;
-		color: var(--text-secondary-color);
+		color: var(--popup-secondary-color);
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/popup.less]: I checked whole guideline to find colors #999999 and #8f8f8f in context of dialog. Only place is calendar - labels for names of days. Could you direct which place in guideline refers to this change?
I think we don't need more variables for the same purpose.

**Comment by TomaszLukawskiSam** [comment]
You should take account our "Contributing code"

#840 
  Rule: " Use your own fork of TAU repository in order to prepare pull request (PR). "

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: You can check it in GUI 1.7 version docs.
“4-5 Dialog_Body text 1 (Title + 2 level description)” on page 108 follows the rules in “14-2 Description_Text” on page 274.
In item 2, Light: #8f8f8f / Dark: #999999 is written.

**Comment by DohyungLim** [comment]
@TomaszLukawskiSam 
ldh@ldh-linux:~/project/public/TAU$ git remote -v
origin	https://github.com/DohyungLim/TAU.git (fetch)
origin	https://github.com/DohyungLim/TAU.git (push)
upstream	https://github.com/Samsung/TAU.git (fetch)
upstream	https://github.com/Samsung/TAU.git (push)

I am making a pull request from pork TAU git. Is there a problem?

**Comment by HunseopJeong** [comment]
image
I agree with @TomaszLukawskiSam,
@DohyungLim, How to input the command when uploading a commit?

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/popup.less]: Ok. Thank you for explanation. I have only one comment. I think that name of variable should be: --popup-text-secondary-color

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**